kubernetes/pod_exec.go

```diff
@@ -38,7 +38,7 @@ func PodExec(pod BackupPod, log logr.Logger) (*ExecData, *bytes.Buffer, error) {
 		SubResource("exec")
 	scheme := runtime.NewScheme()
 	if err := apiv1.AddToScheme(scheme); err != nil {
-		return nil, nil, fmt.Errorf("can't add runtime scheme: %v", err)
+		return nil, fmt.Errorf("can't add runtime scheme: %v", err)
```

Comment:
Use `%w` when wrapping errors. See https://blog.golang.org/go1.13-errors for details